### ISSUE TITLE
BookPlayer stops playing file

### ISSUE THREAD

**Issue by aosullivan1977** [TortugaPower/BookPlayer#1105]
## Environment

 - OS Version: [ios17.3.1]
 - App Version: [5.2.2]

## Description

I downloaded BookPlayer a week ago. And it was working fine. Loaded pools from finder and imported well.

I have just added a couple more books and now the new ones work fine, but none of the old ones work. 
the files are still there, but when you click the play button, it immediately goes back to pause. 

I have restarted the app and my phone. 
I have tried on the phone and through my watch but no good. 

## Steps to reproduce the behavior
1. 
2. 
3.

**Comment by GianniCarlo** [comment]
@aosullivan1977 can you share one of the files that no longer work? also, can you test jumping to the start of one of those books and see if it plays? there are times where the encoded data does not work well with the native libraries that Apple provides, and the asset we get back and load in the native player, for some reason won't have a duration, and the `AVPlayer` just doesn't know what to do with those cases

**Comment by oxonian2019** [comment]
I have the same issue uninstalling and installing the app, erasing the book and its bookmarks does not solve the issue

**Comment by GianniCarlo** [comment]
@oxonian2019 could you share one of the books?

**Comment by oxonian2019** [comment]
Of course. The book is “Essence of Buddhism/ An Introduction to Its Philosophy and Practice.aax” was downloaded from Audible. Neither uninstalling the app and re uploading the book, nor resetting the iPhone worked. The book worked fine for a month of listening with just one other time having this issue before, but this time the app never play again the file. Cheers On 20-11-2024, at 01:31, Gianni Carlo ***@***.***> wrote:﻿
@oxonian2019 could you share one of the books?

**Comment by GianniCarlo** [comment]
@oxonian2019 aaah that makes sense, all the .aax files have DRM, meaning that they should only be playable from Audible (in this case), I'm surprised it plays at all in the app to be honest. If you own the book, there are tools to remove the DRM, so you get the underlying m4b file back that can be played anywhere. 

There's a pretty nifty tool that is [open source](https://github.com/audible-tools/audible-tools.github.io), and they have [the website](https://audible-tools.kamsker.at/) up, and it's super easy to use, you just drag and drop the .aax file, and follow the prompts, and at the end of the process, it will download the m4b file
